### PR TITLE
use state assignment

### PR DESCRIPTION
**Pull request by Jitta-Sri-Vidhya**
use state assignment

### CODE REVIEW

File: src/Amount/use.tsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    const handleAmount = (event: Event, newValue: number| number[]) => {
+        setAmount(newValue as number);
+    };
+    
+    return (
+    <Paper sx={{
+        height:'350px',
+        width:'800px',
+        margin: '10% 20%',
+        padding:'10px 15px 10px 15px'
```
Comment (kamalabhishek) [inline src/Amount/use.tsx:26]: Try to avoid inline styling. Check about 'styled' component in mui.

File: src/Amount/use.tsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    }} elevation={2}>
+
+        <Typo Variant='h6' FontFamily='Roboto'>Amount Details</Typo>
+
+        <Paper sx={{
+            margin: '20px 20px 1px 20px',
+            padding:'15px 20px',
+            display: 'flex',
+            gap:62,
```
Comment (kamalabhishek) [inline src/Amount/use.tsx:35]: Try to remove inline styling

File: src/Amount/use.tsx [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+        }} elevation={2}>
+
+            <Typo FontWeight='bold' Variant="h6">`${amount}`</Typo>
+            <ButtonMui variant="outlined">Buy max</ButtonMui>
+
+        </Paper>
+
+        <Paper sx={{
+            height:'100px',
+            padding:'0px 50px'
```
Comment (kamalabhishek) [inline src/Amount/use.tsx:46]: Same as above, avoid inline styling. Give all the properties like height, padding,margin, gap etc in the styled component in mui.

File: src/Amount/use.tsx [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+            </ThemeProvider>
+
+        </Paper>
+
+        <Paper sx={{
+            margin: '2px 20px 20px 20px',
+            padding:'20px 20px',
+            display: 'flex',
+            gap:70,
+            flexDirection: 'row',
```
Comment (kamalabhishek) [inline src/Amount/use.tsx:69]: Avoid inline styling

**Review by kamalabhishek — CHANGES_REQUESTED** [review]

File: src/Amount/use.tsx (hunk 1 above)
Comment (Jitta-Sri-Vidhya) [reply src/Amount/use.tsx]: i have changed to to the styled component as requested

File: src/Amount/use.tsx (hunk 2 above)
Comment (Jitta-Sri-Vidhya) [reply src/Amount/use.tsx]: removed the inline styling and used the styled component

File: src/Amount/use.tsx (hunk 4 above)
Comment (Jitta-Sri-Vidhya) [reply src/Amount/use.tsx]: removed the inline styling

**Review by kamalabhishek — APPROVED** [review]

**Review by kamalabhishek — APPROVED** [review]

**Pull request merged**